charts/merlin/values.yaml

```diff
@@ -5,7 +5,7 @@ deployment:
     pullPolicy: IfNotPresent
     registry: ghcr.io
     repository: caraml-dev/merlin
-    tag: 0.28.0-1-g07b5aaa
+    tag: 0.0.0-994e9ad6cbed6ac6bfde8b64239318b6c93e3e04
```

Comment:
Kinda caught in between a rock and a hard place due to the app changes that are implemented her in https://github.com/caraml-dev/merlin/pull/397, but cannot be merged and published because it requires a helm-chart upgrade to work with the new config file that's expected. Similarly, the CICD pipeline of this repo cannot pass unless we already have a pre-published updated Merlin app image that we are intending to merge into the `main` branch. 

In the end, I just decided to publish the latest image of Merlin by commenting out the e2e tests in its repo, and then using that tag here in this `values.yaml` file.

Reply:
Yes, for the time-being, this is how breaking changes between the chart and the app are expected to the handled.